src/Sylius/Bundle/ShopBundle/Resources/config/routing.yml

```diff
@@ -23,3 +23,10 @@ sylius_shop_security:
 sylius_shop_checkout:
     prefix: /checkout
     resource: "@SyliusShopBundle/Resources/config/routing/checkout.yml"
+
+sylius_shop_taxon:
+    resource: "@SyliusShopBundle/Resources/config/routing/taxon.yml"
+
+sylius_shop_partial_product:
```

Comment:
All partials should go to separate file, just like in admin. Also you can have a look at #5138.